interface/src/app/gene/resource.js

```diff
@@ -1,22 +1,29 @@
-angular.module('adage.gene.resource', ['ngResource'])
+angular.module('adage.gene.resource', [
+  'ngResource',
+  'adage.utils'
+])
 
-.factory('Gene', ['$resource', function($resource) {
+.factory('Gene', ['$resource', 'ApiBasePath', function($resource, ApiBasePath) {
   return $resource(
-    '/api/v0/gene/:id', {
+    ApiBasePath + 'gene/:id', {
       id: '@id'
     }, {
-
+      post: {
+        url: ApiBasePath + 'gene/',
```

Comment:
Hmm, I don't know if we need to define the url again here since we already defined it (and it should be the same as the one) in line 8. I think the ``/:id`` part should not matter if we do not send the ``id`` as a parameter to this ``post`` action.